apps/frontend/src/libs/components/input/input.tsx

```diff
@@ -5,35 +5,76 @@ import {
 	type FieldValues,
 } from "react-hook-form";
 
-import { useFormController } from "~/libs/hooks/hooks.js";
+import { useCallback, useFormController } from "~/libs/hooks/hooks.js";
 
 type Properties<T extends FieldValues> = {
+	className: string;
+	clickIcon?: (event_: React.BaseSyntheticEvent) => void;
```

Comment:
how about calling it handleIconClick? Feels more descriptive that way

Reply:
yeah, this more better descriptive. thank you